docs/Reference/Request.md

```diff
@@ -27,6 +27,7 @@ Request is a core Fastify object containing the following fields:
 - `protocol` - the protocol of the incoming request (`https` or `http`)
 - `method` - the method of the incoming request
 - `url` - the URL of the incoming request
+- `routeBodyLimit` - either server limit or route limit
```

Comment:
Move Under `routeConfig` would be more consistence.

Reply:
done 👍